src/base.ts

```diff
@@ -19,40 +19,44 @@ export const gasOptions = {
   ...enumEntriesDupWithLowercase(Object.entries(StableToken)),
 }
 
+export const nodeFlagOptions = {
+  char: 'n' as AlphabetLowercase,
+  description: "URL of the node to run commands against (defaults to 'http://localhost:8545')",
```

Comment:
not that this is new, but i notice that while it can take a URL the description doesnt make it obvious that special strings like "local", "alfajores", etc can be passed in too

Reply:
description: "A URL of the node to run commands against (defaults to 'http://localhost:8545') or network / node name. e.g. 'alfajores', 'baklava', 'forno',